client/dashboard/profile-wizard/steps/benefits/index.js

```diff
@@ -153,7 +123,7 @@ class Start extends Component {
 					'Jetpack automatically blocks brute force attacks to protect your store from unauthorized access.',
 					'woocommerce-admin'
 				),
-				visible: ! isJetpackConnected,
+				visible: ! activePlugins.includes( 'jetpack' ),
```

Comment:
Since this logic is repeated a few times in this method, should we just assign the value to a variable and use it instead? `const isJetpackActive = activePlugins.includes( 'jetpack' )`

Reply:
> when did you last do the merge from the 1.0 branch?

Last week's is waiting for @psealock to give it a second look.

Reply:
Updated this in https://github.com/woocommerce/woocommerce-admin/pull/3739/commits/775cac3c3fa5ce2df3b373a8d91cdab463c477f5